x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/logic/export/get_export_by_object_ids.ts

```diff
@@ -93,58 +82,62 @@ export const getExportByObjectIds = async (
   };
 };
 
-export const getRulesFromObjects = async (
+const getRulesFromObjects = async (
```

Comment:
nit: Not sure if the name suits this function anymore after refactoring. As I understand, it's called `getRulesFromObjects` because there was a parameter called "objects" from which rule IDs were read. Perhaps a long time ago whole rule objects were passed into this function, not sure. Since we are passing only an array of IDs now, perhaps it makes sense to rename it to something like `getRulesForIds`. Also not a perfect name, so I don't insist :)

Reply:
Good point. I made this function internal because it's not used anywhere and hesitated to rename it. It's finally renamed to a more descriptive name.